# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/RTCRtpReceiver.h
# Commit: a1c16b86d3a7
# Full Hash: a1c16b86d3a7f328a690c00f86b837a8a67a656c
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Remove RtcpEventObserver. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102287
# ==============================================================================

diff -r c7dbdd82e8e8 -r a1c16b86d3a7 dom/media/webrtc/jsapi/RTCRtpReceiver.h
--- a/dom/media/webrtc/jsapi/RTCRtpReceiver.h	Mon Feb 01 09:28:23 2021 +0100
+++ b/dom/media/webrtc/jsapi/RTCRtpReceiver.h	Mon Jan 18 16:41:26 2021 +0100
@@ -13,7 +13,6 @@
 #include "nsTArray.h"
 #include "mozilla/dom/RTCStatsReportBinding.h"
 #include "RTCStatsReport.h"
-#include "libwebrtcglue/RtcpEventObserver.h"
 #include <vector>
 
 class nsPIDOMWindowInner;
@@ -32,9 +31,7 @@
 struct RTCRtpContributingSource;
 struct RTCRtpSynchronizationSource;
 
-class RTCRtpReceiver : public nsISupports,
-                       public nsWrapperCache,
-                       public RtcpEventObserver {
+class RTCRtpReceiver : public nsISupports, public nsWrapperCache {
  public:
   explicit RTCRtpReceiver(nsPIDOMWindowInner* aWindow, bool aPrivacyNeeded,
                           const std::string& aPCHandle,
@@ -97,10 +94,6 @@
   // This is called when we set a remote description; may be an offer or answer.
   void UpdateStreams(StreamAssociationChanges* aChanges);
 
-  void OnRtcpBye() override;
-
-  void OnRtcpTimeout() override;
-
   void SetReceiveTrackMuted(bool aMuted);
 
  private: